### ISSUE TITLE
Problem with You must use Ember.set() to set the `parentView` property to `null`

### ISSUE THREAD

**Issue by dimul** [piceaTech/ember-rapid-forms#134]
I have updated Ember to ember 2.8 ember-cli to 2.8 and ember-data to 2.8

Now I have a problem with this error

    Uncaught Error: Assertion Failed: You must use Ember.set() to set the `parentView` property (of 
    <myapp@component:em-input::ember619>) to `null`

If I will uncomment this em-inputs in my template, all works just fine

It happens when I am transitioning from a particular route to another. Whereas when going straight into route that do not have em-template in it and try to transition to different routes, all works fine.

---
`ember version --verbose`

```
ember-cli: 2.8.0
http_parser: 2.7.0
node: 6.5.0
v8: 5.1.281.81
uv: 1.9.1
zlib: 1.2.8
ares: 1.10.1-DEV
icu: 57.1
modules: 48
openssl: 1.0.2h
os: darwin x64

```

**Comment by GCorbel** [comment]
Hi, I know that issue and I prepared a fix. I will push it today.

The error comes from this file : https://github.com/piceaTech/ember-rapid-forms/blob/master/addon/mixins/in-form.js .You can just remove `parentView` from the computed property.

**Comment by dimul** [comment]
Ok, got you, thx for update

There is actually an update in emberjs/ember.js

There was a fix introduced. Check this pull-request which was alredy merged... quite recently actually emberjs/ember.js#14281

It actually fixes it. Just finished testing

**Comment by spruce** [comment]
Is this resolved then?

**Comment by dimul** [comment]
No man, it is still not. Stuff needs fixing and without it it seems that it is not possible to use Ember 2.8. At least for us here

**Comment by dimul** [comment]
This is what continues to happen, after applying a fix from mentioned pull request 
```
ember.debug.js:6778 Uncaught Error: Assertion Failed: You must use Ember.set() 
to set the `parentView` property (of <appname@component:em-num-input::ember1061>) 
to `<appname@component:em-form::ember930>`.
```

**Comment by GCorbel** [comment]
I should be solved by this PR : https://github.com/piceaTech/ember-rapid-forms/pull/135 . @spruce , can you check it and tell me if it's correct ?

**Comment by dimul** [comment]
@GCorbel I have checked it and it works for me in all those places that were not working

**Comment by spruce** [comment]
Nice. So we can close this issue when we have merged the PR? @dimul ?

**Comment by dimul** [comment]
Yeap, it seems like so

**Comment by spruce** [comment]
Closed as the PR (#135) is merged.
